yammbs/torsion/_store.py

```diff
@@ -368,7 +370,21 @@ def get_rmsd(
                 allow_undefined_stereo=True,
             )
 
-            rmsds.append(RMSD.from_data(torsion_id, molecule, qm_points, mm_points))
+            rmsds.append(
+                RMSD(
+                    id=torsion_id,
+                    rmsd=sum(
+                        get_rmsd(
+                            molecule,
+                            qm_points[key],
+                            mm_points[key],
+                            include_hydrogens=include_hydrogens,
+                        )
+                        for key in qm_points
+                    )
+                    / len(qm_points),
```

Comment:
Note this detail - without which there were numeric differences, i.e. 24x for a scan of 24 grid points